### PR TITLE
[HDFS] Add ZKFC to NameNodes

### PR DESCRIPTION
**Pull request by NimaVaziri**
Enables fully HA deployment via enabling:
 - Failure detection
 - Active NameNode election

Namenode-0: `16/11/01 00:01:07 INFO ha.ZKFailoverController: Successfully transitioned NameNode at namenode-0.hdfs.mesos/10.0.2.168:9001 to active state`

Namenode-1: `16/11/01 00:01:27 INFO ha.ZKFailoverController: ZK Election indicated that NameNode at namenode-1.hdfs.mesos/10.0.2.169:9001 should become standby`

### CODE REVIEW

File: frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java [hunk 1]
```diff
@@ -185,18 +185,24 @@ private static String addNameNodeBootstrapCommands(int instanceIndex) {
         if (instanceIndex == 0) {
             return String.format("echo 'Formating filesystem';" +
                     "./%s/bin/hdfs namenode -format -force && " +
+                    "echo 'ZKFC format';" +
+                    "./%s/bin/hdfs zkfc -formatZK -force && " +
                     "echo 'Initialize Shared Edits' && " +
                     "./%s/bin/hdfs namenode -initializeSharedEdits -force && " +
+                    "echo 'Starting ZKFC process';" +
+                    "./%s/bin/hdfs zkfc & " +
```
Comment (NimaVaziri) [inline frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java]: Note that both ZKFC and NameNode processes need to be running in parallel, hence sending the `zkfc` process to the background.

File: frameworks/hdfs/build.sh [hunk 2]
```diff
@@ -9,6 +9,10 @@ cd $REPO_ROOT_DIR
 # Grab dcos-commons build/release tools:
 rm -rf dcos-commons-tools/ && curl https://infinity-artifacts.s3.amazonaws.com/dcos-commons-tools.tgz | tar xz
 
+_notify_github() {
+  GIT_REPOSITORY_ROOT=$ROOT_DIR $ROOT_DIR/tools/github_update.py $1 build:reference $2
```
Comment (mohitsoni) [inline frameworks/hdfs/build.sh]: Where is `ROOT_DIR` defined ? The script defines `REPO_ROOT_DIR` on Line 5

Comment (NimaVaziri) [inline frameworks/hdfs/build.sh]: Wasn't defined as that wasn't the intended change, refer to the latest version.

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Pull request closed without merging**